### PR TITLE
adding build recipe for phyde-0.3.2 [skip appveyor]

### PR DESCRIPTION
**Pull request by pblischak**

### CODE REVIEW

File: recipes/phyde/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win or (not py27)]
+  number: 2
```
Comment (CJ-Wright) [inline recipes/phyde/meta.yaml]: You can set this to 0 in staged-recipes

Comment (pblischak) [reply recipes/phyde/meta.yaml]: Gotcha -- will do. Thanks for letting me know!

File: recipes/phyde/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - cython
+    - pandas
+    - numpy
+    - matplotlib
+    - seaborn
+    - multiprocess
+
+test:
+  imports:
+    - phyde
```
Comment (CJ-Wright) [inline recipes/phyde/meta.yaml]: can you please include some more import tests?

Comment (pblischak) [reply recipes/phyde/meta.yaml]: I just added some more import tests. Please let me know if you want me to add any others.

Comment (CJ-Wright) [reply recipes/phyde/meta.yaml]: That looks good!

**Pull request closed without merging**